server/src/main/java/org/apache/cassandra/sidecar/acl/authentication/JwtParameters.java

```diff
@@ -52,4 +53,44 @@ public interface JwtParameters
      * dynamically retrieve configuration information of an OpenID provider.
      */
     SecondBoundConfiguration configDiscoverInterval();
+
+    /**
+     * @return The configured method of JWT authentication to use. Defaults to oauth if not supplied.
+     */
+    AuthType jwtAuthType();
+
+    /**
+     * @return Optional path to a keystore to provide mutual TLS to PEM public key provider service
+     */
+    Optional<String> keystorePath();
```

Comment:
I think generally we discourage usage of Optional in the project as the return type of a method, and instead perform the null check or surface things with exceptions instead. Looking at the usage of this method

```
if (jwtParameters.keystorePath().isPresent())
```

can be easily changed to 

```
if (jwtParameters.keystorePath() != null)

Reply:
Thank you @frankgh. Addressed in https://github.com/apache/cassandra-sidecar/pull/247/commits/ef5e1ef5ad100bc5798ba254c8619b21ee077b57